### ISSUE TITLE
Unsafe / raw data inside xml

### ISSUE THREAD

**Issue by krzysztofsroga** [redundent/kotlin-xml-builder#53]
Hi! is there any way to insert raw data inside xml?
Kotlin HTML dsl has `unsafe{}` block that allows me to insert raw data. Xml builder however, does not seem to have any option to skip escaping characters.

For example I'm trying to generate keyboard layouts and I want to set `&#x0010;` as an attribute value

```kotlin
"key" {
    attributes(
        "code" to 108,
        "output" to "&#x0010;"
    )
}
```
The library generates xml:
```xml
<key code="108" output="&amp;#x0010;"/>
```
While I need it to generate:
```xml
<key code="108" output="&#x0010;"/>
```

Any workaround?
If I could just insert whole `<key code="108" output="&#x0010;"/>` tag string as raw, I'd be happy as well but unfortunately it's also escaped.

**Comment by redundent** [comment]
I don't think there's a way to do it in it's current state. I can see about adding an equivalent `unsafe` method to support it.
I can probably get it done on Monday

**Comment by krzysztofsroga** [comment]
Thanks, that sounds great!
For now I made a workaround of keeping a placeholder for all these values and replacing them at the end so I'm not held back by this for now.
Unsafe block seems like a great addition overall!

**Comment by redundent** [comment]
I haven't had a chance to work on this yet. Sorry about that. Things are a bit crazy at work. Hopefully I'll get some time later today or tomorrow

**Comment by redundent** [comment]
Hey @krzysztofsroga , sorry it took so long but this is available in version 1.9.0 which I just published to Maven Central.
Check out the readme for an example.

**Comment by krzysztofsroga** [comment]
I used the new version in my project and it works as expected.
Thanks a lot!
